cypress/e2e/wip/Tradershub/accountSignupMF.cy.js

```diff
@@ -4,7 +4,7 @@ import {generateEpoch} from '../../../support/tradersHub'
 
 describe("QATEST-5569: Verify MF Signup flow", () => {
   const epoch = generateEpoch()
-  const sign_up_mail = `sanity${epoch}MF@deriv.com`
+  const sign_up_mail = `sanity${epoch}mf@deriv.com`
```

Comment:
If epoch is no longer required multiple times we can update this line as 

```suggestion
  const sign_up_mail = `sanity${generateEpoch()}mf@deriv.com`
```

and remove const epoch = generateEpoch()

from all similar cases as well